src/oasis-indexer/api.ts

```diff
@@ -45,29 +50,30 @@ export const useGetEmeraldTransactions: typeof generated.useGetEmeraldTransactio
       ],
     },
   })
-  return result
 }
 
-export const useGetConsensusAccountsAddress: typeof generated.useGetConsensusAccountsAddress = (
+export const useGetRuntimeAccountsAddress: typeof generated.useGetRuntimeAccountsAddress = (
+  runtime = Runtime.emerald,
   params,
   options?,
 ) => {
-  const result = generated.useGetConsensusAccountsAddress(params, {
+  return generated.useGetRuntimeAccountsAddress(runtime, params, {
     ...options,
     axios: {
       ...options?.axios,
       transformResponse: [
         ...arrayify(axios.defaults.transformResponse),
-        (data: generated.Account) => {
+        (data: generated.RuntimeAccount) => {
           return {
             ...data,
-            runtime_evm_balances: data.runtime_evm_balances?.map(token => {
+
+            runtime_evm_balances: data.evm_balances?.map(token => {
               return {
                 ...token,
                 balance: token.balance ? fromBaseUnits(token.balance, token.token_decimals) : undefined,
               }
             }),
-            runtime_sdk_balances: data.runtime_sdk_balances?.map(token => {
+            runtime_sdk_balances: data.balances?.map(token => {
```

Comment:
Will we migrate to same field names in a separate PR?
```suggestion
            balances: data.balances?.map(token => {
```